src/build123d/topology.py

```diff
@@ -5125,8 +5082,10 @@ def project_to_shape(
 
     def to_axis(self) -> Axis:
         """Translate a linear Edge to an Axis"""
-        if self.geom_type() != "LINE":
-            raise ValueError("to_axis is only valid for linear Edges")
+        if self.geom_type() != GeomType.LINE:
+            raise ValueError(
+                "to_axis is only valid for linear Edges not " + self.geom_type()
```

Comment:
I think `f"to_axis is only valid for linear edges not edges of type {self.geom_type()} "` is better.